source/renderer/app/components/wallet/tokens/wallet-token-picker/helpers.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import xor from 'lodash/xor';
+import { messages } from './WalletTokenPicker.messages';
+import {
+  MAX_TOKENS,
+  ScrollPositionEnum,
+  FilterSelectOptionsEnum,
+} from './const';
+import type { Intl } from '../../../../types/i18nTypes';
+import type {
+  Assets,
+  BooleanMap,
+  FilterAssets,
+  ScrollPosition,
+  GetMaxTokensIdsMap,
+} from './types';
+
+const isScrollAtTop = (element: HTMLElement) => element.scrollTop === 0;
+
+const isScrollAtBottom = (element: HTMLElement) =>
+  element.scrollHeight - element.scrollTop === element.clientHeight;
+
+export const getScrollPosition = (element: EventTarget): ScrollPosition => {
+  if (!(element instanceof HTMLElement)) {
+    return ScrollPositionEnum.TOP;
+  }
+
+  if (isScrollAtTop(element)) {
+    return ScrollPositionEnum.TOP;
+  }
+
+  if (isScrollAtBottom(element)) {
```

Comment:
good use for an else tied to previous block as it can only be one or the other

Reply:
I had an else in there at some point but there was a lint issue due to no-else-return rule